### DISCUSSION TITLE
A question about modpath7

### DISCUSSION THREAD

**Discussion by liufan612** [modflowpy/flopy#1865] [Q&A]
No matter how I modify the modpath7 input file, my modpath model will only run for one stress period and stop。
**The simulation will be run with stoptime =   0.0000000E+00**
I find this sentence a bit weird compared to the example file, which I have in my mplst file。
I think flow model is being read normally，
this is my code and mplst file below.
    flopy.modpath.Modpath7Sim(
        modpath_model,
        simulationtype="combined",
        trackingdirection=tracking_direction,
        weaksinkoption="pass_through",
        weaksourceoption="pass_through",
        budgetoutputoption="summary",
        stoptimeoption="extend", 
        particlegroups=particle_group)
MODPATH name file data
----------------------
   MPBAS File: mp7_nfc.mpbas
 GRBDISV File: gwf.dis.grb
    TDIS File: nfc-mf6.tdis
    HEAD File: gwf.hds
  BUDGET File: gwf.bud

Grid data
---------
Grid file type: MODFLOW-6 DIS binary grid file.
Time discretization file type: MODFLOW-6 time discretization file.
 # File generated by Flopy version 3.3.6 on 07/04/2023 at 11:37:06.
 
 Time discretization file data
 -----------------------------
 Stress period count =         12
 Period length:       28.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       30.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       30.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       30.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       30.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       31.00000 Time step count:         1 Time step multiplier:        1.00000
 Period length:       30.00000 Time step count:         1 Time step multiplier:        1.00000
 
 The budget file was opened successfully.

**Comment by wpbonelli** [comment]
Hello @liufan612, for a simulation with `stoptimeoption="extend"`, the stop time should automatically be set to a very large number, e.g. 1e+30. There may be a configuration error in one of your model input files.

To help identify the issue, could you please provide the complete Python program you are using to construct your model. If you wouldn't mind also pasting the code into a [block surrounded by triple-backticks](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) so it is easier to read.

It may be helpful to know the version of MODPATH 7 you are using, and how you acquired it &mdash; from the [USGS site](https://www.usgs.gov/software/modpath-particle-tracking-model-modflow), or the [executables repository release](https://github.com/MODFLOW-USGS/executables), etc. You might also try upgrading FloPy. Version 3.4.1 was released last week.
